eventually/authentication/models.py

```diff
@@ -83,7 +83,7 @@ def create(first_name, middle_name, last_name, email, password):
             pass
         return user
 
-    def myattr_to_dict(self):
+    def to_dict(self):
         """
         :return: manually-defined attributes of the object. Inherited attributes aren't shown.
         :example: {'id': 8, 'first_name': 'fn', 'middle_name': 'mn',
```

Comment:
`'FIXME` please replace it with `TODO` or remove.

Reply:
it is not my app. I have remoned  eventually/authentication/models.py